ocs/ocs_agent.py

```diff
@@ -739,6 +747,17 @@ def encoded(self):
 
 SESSION_STATUS_CODES = [None, 'starting', 'running', 'stopping', 'done']
 
+
+class OpCode(Enum):
+    NONE = 1
+    STARTING = 2
+    RUNNING = 3
+    STOPPING = 4
+    SUCCEEDED = 5
+    FAILED = 6
+    EXPIRED = 7
```

Comment:
The Enum is potentially of interest to consumers that don't have twisted available... and thus might not want to import ocs_agent.  Can it be moved to "base.py", which is sort of like the header file for ocs?

Reply:
I acknowledge that I did not do this with 'starting', 'running', ...  I should have!